package.json

```diff
@@ -48,7 +48,7 @@
     "node": ">=6"
   },
   "scripts": {
-    "pretest": "eslint .",
+    "pretest": "eslint . --fix",
```

Comment:
The problem is that adding this means the CI will never fail if there's an eslint error on the branch.

Reply:
Rolled back

Reply:
There is a weird error on the travis CI build, it passes for Node 6 but fails for 8 and `current` while running ESLint. Im running the latest version and it passes for me. Any clue of what could be going on?